src/core_modules/capture-core/metaData/RenderFoundation/CustomForm.js

```diff
@@ -34,13 +34,17 @@ export class CustomForm {
      *
      * @memberof CustomForm
      */
-    set data(html: string) {
-        const data = parseHtml(html, {
-            onTransform: this.transformNode,
+    setData(html: string, transformFunction: () => ?React.Element<'FormField'>) {
```

Comment:
NIT: I think it would be a good exercise to improve up on this Flow declaration. (Part 1)